homeassistant/components/glances/__init__.py

```diff
@@ -174,6 +173,4 @@ def get_api(hass, entry):
     """Return the api from glances_api."""
     params = entry.copy()
     params.pop(CONF_NAME)
-    verify_ssl = params.pop(CONF_VERIFY_SSL)
```

Comment:
Existing entries may still have this verify ssl item. We should still pop it, but use a default if missing, I think.
https://docs.python.org/3/library/stdtypes.html#dict.pop

Reply:
So pop it and ignore the value?

Reply:
Yes. Please add a comment too, why we need it.

Reply:
Sure
👍🏻